pkg/sql/sqlbase/structured.go

```diff
@@ -2634,11 +2634,11 @@ func (desc *MutableTableDescriptor) DropConstraint(
 
 	case ConstraintTypeFK:
 		if detail.FK.Validity == ConstraintValidity_Validating {
-			return unimplemented.Newf("drop-constraint-fk-mutation",
+			return unimplemented.NewWithIssuef(42844,
```

Comment:
can you use `WithTelemetry` to differentiate the various FK operations so we get detailed usage telemetry.